modules/042-kube-dns/monitoring/prometheus-rules/kubernetes/dns.yaml

```diff
@@ -15,3 +15,25 @@
         CoreDNS pod {{$labels.pod}} has at least one critical error.
         To debug the problem, look into container logs: `kubectl -n kube-system logs {{$labels.pod}}`
       summary: CoreDNS has critical errors.
+
+- name: deckhouse.kube-dns.legacy-services
+  rules:
+  - alert: D8KubeDnsServiceWithDeprecatedAnnotation
+    expr: d8_kube_dns_deprecated_service_annotation == 1
```

Comment:
I think we need to trim excessive labels here, e.g., by wrapping it in the max function
```
max by (namespace, name) (d8_kube_dns_deprecated_service_annotation) == 1
```

Reply:
Makes sense. Done.